src/utils/validation.ts

```diff
@@ -2,9 +2,8 @@ import chains from '@/config/chains'
 import { parsePrefixedAddress, sameAddress, isChecksummedAddress } from './addresses'
 import { safeFormatUnits, safeParseUnits } from './formatters'
 
+export const ADDRESS_RE = /^0x[0-9a-f]{40}$/i
```

Comment:
Instead of exporting a regex, export a function `isAddress` that would use that regex.